MultisigTests/Logic/Models/TransactionTests.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  TransactionTests.swift
+//  MultisigTests
+//
+//  Created by Dmitry Bespalov on 16.06.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import XCTest
+@testable import Multisig
+
+class TransactionTests: XCTestCase {
+
+    func testAddOwner() {
+        let txJson = """
```

Comment:
I would extract JSONs into a separate file. Otherwise, it is hard to read.

Reply:
Please ignore this, this test is work in progress, I'll do that and other tests in the following PR 🙏